.github/workflows/build.yml

```diff
@@ -1,27 +1,22 @@
 name: build
 
-on: [push, pull_request]
+on: 
+  push:
+    branches: [main]
+  pull_request:
 
 jobs:
   test:
     runs-on: ubuntu-latest
     strategy:
       fail-fast: false
       matrix:
-        ruby: [2.5, 2.6, 2.7, "3.0", 3.1, head]
-        gemfile: [rails_5_2, rails_6]
```

Comment:
Why get rid of 2.5 and rails 5 support?

Reply:
It makes it harder to maintain over time. If you think it makes sense to re-add, I'm happy to.